src/encoder.js

```diff
@@ -9,6 +9,9 @@ const F2 = 0xFF;
 const zeros23 = '00000000000000000000000';
 let keysetLUT, keysetList: Array<Array<string>>, stringHist, stringPlaceholders;
 
+function isNaN(value) { // eslint-disable-line no-shadow
```

Comment:
Shadowing the global `isNaN` is going to make the usage below confusing. Also, why can't we use `Number.isNaN`?